playwright/_impl/_api_structures.py

```diff
@@ -26,6 +26,7 @@
 
 # Explicitly mark optional params as such for the documentation
 # If there is at least one optional param, set total=False for better mypy handling.
+from playwright._impl._devices_generated import DeviceDescriptor, Devices
```

Comment:
Why do we need to import this here? I'd instead define types here, and import them in `_devices_generated.py`. This way we don't need the `__all__` hack. WDYT?

Reply:
If you want to directly import it then you would need some custom handling in docs https://github.com/microsoft/playwright-python/blob/cfc1030a69d1e934cac579687a680eac53d4b9ee/scripts/generate_api.py#L129, also __all__ is not a hacky solution, it is there to not make falke8 complain about not exported imports

Reply:
If it's only flake8 there which is complaining then I would keep it as it was, since we don't have the flake8 linter in place and it was working before.

Reply:
Are you sure that we don't have falke8 https://github.com/microsoft/playwright-python/blob/835cdce3f0497192748a806bfe537051f1dd18d2/.pre-commit-config.yaml#L20 ?

Reply:
Ignored now